egg/zoo/compo_vs_generalization/train.py

```diff
@@ -252,7 +259,10 @@ def main(params):
     else:
         raise ValueError(f"Unknown sender cell, {opts.sender_cell}")
 
-    sender = PlusOneWrapper(sender)
+    print(opts.fixed_length)
```

Comment:
Why not print `opts` directly? this way you can inspect the value of all parameters